.github/create-cli-pr.sh

```diff
@@ -14,30 +15,37 @@
 # limitations under the License.
 #
 
-#!/bin/bash
 set -ex
 
 CLI_DIR=$(mktemp -d)
-gh repo clone git@github.com:snyk/cli.git $CLI_DIR -- --depth=1
-pushd $CLI_DIR
+
+
+gh repo clone git@github.com:snyk/cli.git "$CLI_DIR" -- --depth=1
+pushd "$CLI_DIR/cliv2"
+  LS_COMMIT_HASH=$(grep snyk-ls go.mod| cut -d "-" -f 4)
+popd
+
+WHAT_CHANGED=$(git whatchanged "$LS_COMMIT_HASH"...HEAD)
```

Comment:
should we use git log instead of git whatchanged? 
> The command is kept primarily for historical reasons;

https://stackoverflow.com/a/18585297
according to this answer equivelant should be 
```git log --raw --no-merges```

Reply:
Interesting. I like `whatchanged` better, as it's quite explicit :)